frontend/src/scenes/session-recordings/saved-playlists/savedSessionRecordingPlaylistsLogic.ts

```diff
@@ -205,6 +206,29 @@ export const savedSessionRecordingPlaylistsLogic = kea<savedSessionRecordingPlay
             },
         ],
     })),
+    listeners(() => ({
+        loadPlaylistsSuccess: ({ playlists }) => {
+            try {
```

Comment:
wrapped in a try catch because this code absolutely shouldn't cause user problems if there's something unexpected here